src/mumble/LookConfig.ui

```diff
@@ -621,6 +621,23 @@
         </property>
        </widget>
       </item>
+      <item row="13" column="2">
+       <widget class="QSpinBox" name="qsbTransparencyLevel">
+        <property name="minimum">
+         <number>0</number>
+        </property>
+        <property name="maximum">
+         <number>255</number>
+        </property>
+    
+        <property name="toolTip">
+         <string>Transparency level of the TalkingUI. </string>
+        </property>
+        <property name="accessibleName">
+         <string>Transparency level</string>
+        </property>
```

Comment:
This is a bad ``accessibleName``. This field is used for users with assistive technology such as screen readers. When they tab into this new field they will simply hear "transparency level" and there is no context that this affects the talking UI.

I'd change this and the ``toolTip`` to "TalkingUI background transparency level" or something similar